docs/zh/latest/plugins/skywalking.md

```diff
@@ -1,5 +1,11 @@
 ---
 title: skywalking
+keywords:
+  - APISIX
+  - Plugin
+  - SkyWalking
+  - skywalking
```

Comment:
What is the difference between them?

Reply:
We are not sure which keywords users will search by, so we use keywords with different capitalization. It will be optimized based on SEO in the future.

Reply:
We are currently unsure which keywords will be helpful for this document. So keep it consistent. https://github.com/apache/apisix/pull/6883#pullrequestreview-949748616

Reply:
![image](https://user-images.githubusercontent.com/2106987/172276575-22cc30ef-7445-41e9-bd6c-05618461d021.png)

Please just keep using `SkyWalking`, the same to other documents.